MakerTracker/DBModels/Transaction.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace MakerTracker.DBModels
+{
+    public class Transaction
```

Comment:
Also I don't see the migration for this table in the PR.

Reply:
You're right, missed it in staging and didn't notice since I didn't test the migrations right before PR